chcemvediet/locale/sk/LC_MESSAGES/django.po

```diff
@@ -50,6 +50,9 @@ msgstr "PSČ"
 msgid "accounts:SignupForm:agreement:label"
 msgstr "Súhlasím s podmienkami používania."
 
+msgid "accounts:SettingsForm:anonymized_inforequest:label"
+msgstr "Anonymizovanie zverejnených infožiadostí"
```

Comment:
Skor to formuluj "Zverejnené infožiadosti zverejniť anonymizovane" alebo tak nejak.